specification/vision/ImageAnalysis/tspconfig.yaml

```diff
@@ -0,0 +1,34 @@
+parameters:
+  "service-dir":
+    default: "sdk/vision"
+emit:
+ - "@azure-tools/typespec-autorest"
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-core/all"
+options:
+  "@azure-tools/typespec-autorest":
+    emitter-output-dir: "{project-root}/../"
+    output-file: "{azure-resource-provider-folder}/ImageAnalysis/{version-status}/{version}/generated.json"
+    azure-resource-provider-folder: "data-plane"
+    examples-directory: examples
+    omit-unreachable-types: true
+  "@azure-tools/typespec-python":
+    package-dir: "azure-ai-vision-imageanalysis"
+    package-name: "{package-dir}"
+  "@azure-tools/typespec-csharp":
+    package-dir: "Azure.AI.Vision.ImageAnalysis"
+    namespace: "Azure.AI.Vision.ImageAnalysis"
+    clear-output-folder: true
+    model-namespace: false
+  "@azure-tools/typespec-java":
+    package-dir: "azure-ai-vision-imageanalysis"
+    namespace: "com.azure.ai.vision.imageanalysis"
+    # We switched two public methods to private. Make sure this is not overridden next time the SDK is emitted.
+    partial-update: true
+  "@azure-tools/typespec-ts":
+    package-dir: "imageAnalysis"
+    generateMetadata: true
+    generateTest: true
```

Comment:
Could we remove below settings? Codegen would automatically detect if we need to generte metadata and we don't need to set them explicitly.

```
    generateMetadata: true
    generateTest: true
```